src/main.rs

```diff
@@ -77,7 +81,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 if let Err(err) = update_next_block(&server).await {
                     eprintln!("update_next_block error: {:?}", err);
                 }
-                tokio::time::sleep(Duration::from_secs(3)).await;
```

Comment:
Why increase this? Maybe just make it a config variable as well, e.g. `rpcupdateinterval` if you're changing it anyway